torch/optim/adam.py

```diff
@@ -253,7 +253,8 @@ def step(self, closure=None):
        \begin{aligned}
             &\rule{110mm}{0.4pt}                                                                 \\
             &\textbf{input}      : \gamma \text{ (lr)}, \beta_1, \beta_2
-                \text{ (betas)},\theta_0 \text{ (params)},f(\theta) \text{ (objective)}          \\
+                \text{ (betas)},\theta_0 \text{ (params)},f(\theta) \text{ (objective)}
```

Comment:
Can you move this to after the maximize?

Reply:
Can you also include a docs preview when you get the chance?

Reply:
Hi @janeyx99 I've made the requested change. Just waiting for approval on the CI pipeline before the docs update. Here is the docs preview on the original version of the PR. 
![image](https://github.com/user-attachments/assets/a280e520-f59c-47c7-95f9-6ed439518293)